Examples/Swift/PointHeatmapExample.swift

```diff
@@ -0,0 +1,54 @@
+
+import Mapbox
+
+@objc(PointHeatmapExample_Swift)
+
+class PointHeatmapExample_Swift: UIViewController, MGLMapViewDelegate {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let mapView = MGLMapView(frame: view.bounds, styleURL: MGLStyle.darkStyleURL(withVersion: 9))
+        mapView.delegate = self
+        
+        view.addSubview(mapView)
+    }
+    
+    func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle) {
+        
+        // Parse GeoJSON data from USGS on earthquakes in the past week.
+        DispatchQueue.global().async {
+            guard let url = URL(string: "https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/all_week.geojson") else { return }
+            DispatchQueue.main.async {
+                self.displayEarthquakes(url: url, style: style)
+            }
+        }
+    }
+    
+    func displayEarthquakes(url: URL, style: MGLStyle) {
+        let symbolSource = MGLSource(identifier: "symbol-source")
+        let symbolLayer = MGLSymbolStyleLayer(identifier: "place-city-sm", source: symbolSource)
+        
+        // Set the MGLShapeSourceOptions to allow clustering.
+        let options = [MGLShapeSourceOption.clustered: true,
+                       MGLShapeSourceOption.clusterRadius: 20,
+                       MGLShapeSourceOption.maximumZoomLevel: 15] as [MGLShapeSourceOption : Any]
+        
+        let earthquakeSource = MGLShapeSource(identifier: "earthquakes", url: url, options: options)
```

Comment:
I changed the source to a Mapbox hosted one. Will look into using this live data source for a tutorial/different example.